composer.lock

```diff
@@ -4865,24 +4865,24 @@
         },
         {
             "name": "oat-sa/extension-tao-task-queue",
-            "version": "v6.7.2",
+            "version": "v6.8.2",
```

Comment:
Could you list all changes before the minor release? We should backport instead of bumping minor version.